README.md

```diff
@@ -6,6 +6,12 @@ Uploads continue in the background, even after a page transition. In other words
 
 [View the docs here.](https://adopted-ember-addons.github.io/ember-file-upload/docs/)
 
+## Compatibility
+
+Ember.js v2.18 or above
+Ember CLI v3.5.1 or above
```

Comment:
> The ember-cli version I chose is just the version from package.json not sure what we actually support.

As we don't do anything special in the build pipeline of this addon, going with defaults of Ember CLI blueprint should be fine:

```suggestion
Ember CLI v2.13 or above
```

Reply:
👍